src/Product/ProductRepository.php

```diff
@@ -357,6 +357,8 @@ protected function prepare_query_args( array $args = [] ): array {
 			$args['orderby'] = 'none';
 		}
 
+		$args = apply_filters('gla_filter_product_query_args', $args);
```

Comment:
Can we use the same naming as the rest of the hooks?

`woocommerce_gla_product_query_args`